DH_params:
  theta: [0, 0, 0, 0, 0, 0] 
  alpha: [90, 0, 0, -90, 90, 0]
  a: [0, 0.269, 0.34516, 0.34516, 0.27, 0.269]
  d: [0.20271, 0, 0, 0, 0, 0]

IK_solver: 
  tol: 0.00001
  max_iter: 1000